src/librepgp/stream-ctx.h

```diff
@@ -121,6 +121,12 @@ void             rnp_ctx_reset(rnp_ctx_t *);
 void             rnp_ctx_free(rnp_ctx_t *);
 struct rng_st_t *rnp_ctx_rng_handle(const rnp_ctx_t *ctx);
 
+rnp_result_t rnp_encrypt_set_pass_info(rnp_symmetric_pass_info_t *info,
+                                       const char *               password,
+                                       pgp_hash_alg_t             hash_alg,
+                                       size_t                     iterations,
+                                       pgp_symm_alg_t             s2k_cipher);
+
```

Comment:
This function should be just removed, looks like was used previously and superseded by `rnp_ctx_add_encryption_password()`.